Bridges/HttpKernel.php

```diff
@@ -240,12 +238,22 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
 
         $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
 
-        $stdOut = '';
-        while ($buffer = @ob_get_clean()) {
-            $stdOut .= $buffer;
+        // asynchronously get content
+        ob_start(function($buffer) use ($reactResponse) {
+            $reactResponse->write($buffer);
```

Comment:
I have no idea why, but this line makes my whole response body empty, with a not streamed response 😟